Package.swift

```diff
@@ -136,6 +136,18 @@ let package = Package(
             name: "TuistCacheIntegrationTests",
             dependencies: ["TuistCache", "TuistSupportTesting", "RxBlocking", "TuistCoreTesting"]
         ),
+        .target(
+            name: "TuistTemplate",
```

Comment:
Nitpick: What about calling this one `TuistScaffold`?

Reply:
Good point, renamed.